packages/blocks/src/store/selectors.js

```diff
@@ -789,3 +789,13 @@ export const hasChildBlocksWithInserterSupport = ( state, blockName ) => {
 		return hasBlockSupport( state, childBlockName, 'inserter', true );
 	} );
 };
+
+export const __unstableIsContentBlock = createSelector(
```

Comment:
What do you think of expanding the existing naming ([example](https://github.com/WordPress/gutenberg/blob/e09fc6ee259385edefc0ae0e14eed82775cba57b/packages/blocks/src/api/utils.js#L293)) and use something along the lines of `__experimentalHasContentRoleAttribute`? As the first time I ran into this, it was not clear what a "content block" was.